packages/cli-helpers/src/auth/__tests__/setupHelpers.test.ts

```diff
@@ -28,42 +28,45 @@ jest.mock('../../lib/paths', () => {
   }
 })
 
-jest.mock('../../lib/project', () => ({
+vi.mock('../../lib/project', () => ({
   isTypeScriptProject: () => true,
 }))
 
-jest.mock('execa', () => {})
-jest.mock('listr2')
-jest.mock('prompts', () => jest.fn(() => ({ answer: true })))
+vi.mock('execa')
+vi.mock('listr2')
+vi.mock('prompts', () => ({
+  default: vi.fn(() => ({ answer: true })),
+}))
 
 import fs from 'fs'
 import path from 'path'
 
 import { Listr } from 'listr2'
 import prompts from 'prompts'
+import { vi, describe, afterEach, it, expect } from 'vitest'
 
 // import * as auth from '../auth'
 import { standardAuthHandler } from '../setupHelpers'
 
 describe('Auth generator tests', () => {
-  const processExitSpy = jest
+  const processExitSpy = vi
     .spyOn<NodeJS.Process, any>(process, 'exit')
     .mockImplementation((_code: any) => {})
 
-  const mockListrRun = jest.fn()
+  const mockListrRun = vi.fn()
 
-  ;(Listr as jest.MockedFunction<jest.Mock>).mockImplementation(() => {
+  ;(Listr as vi.MockedFunction<vi.Mock>).mockImplementation(() => {
```

Comment:
In the future, when someone cleans up our testing, this should be fixed. There are much better solutions than type-casting to use here.

Nothing to do now